src/olympia/lib/git.py

```diff
@@ -66,17 +66,18 @@ class AddonGitRepository(object):
     def __init__(self, addon_id, package_type='package'):
         assert package_type in ('package', 'source')
 
-        # Always enforce the search path being set to the configured
-        # HOME environment variable. We are setting this here to avoid
-        # creating a unnecessary global state but since this is overwriting
-        # a global value in pygit2 it affects all pygit2 calls.
-        # Given that we're setting it to whatever `~` expands to
-        # the actual value changes more or less only on process-restart.
+        # Always enforce the search path being set to our GIT_STORAGE_ROOT
```

Comment:
`GIT_FILE_STORAGE_PATH` not `GIT_STORAGE_ROOT` ? Also, I wonder if we should choose a different setting, to avoid libgit searching for non-existing files on a remote filesystem.

Reply:
fair point, I wonder if setting this to `/tmp/` might even be a reasonable idea?

Reply:
tbh... maybe just `settings.ROOT` would be enough...

Reply:
I like that, fixed in cae524101d